packages/react-proptypes/src/editorOptions.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    autoparagraph: PropTypes.bool,
+    placeholder: PropTypes.string,
+    formatting: PropTypes.shape({
+        strong: PropTypes.bool,
+        b: PropTypes.bool,
```

Comment:
`b` and `i` don't work with CKE I think (`strong` and `em` instead). If so then should we provide a migration or just leave as is and remove from the schema?

Reply:
Is this solved with the current code? Then we could just drop them here?